react/src/components/atoms/forms/InputTextFuzzy/index.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Fuse from 'fuse.js';
+import Autowhatever from 'react-autowhatever';
+import parse from 'autosuggest-highlight/parse';
+import classNames from 'classnames';
+
+import './style.css';
+
+class InputTextFuzzy extends React.Component {
```

Comment:
the suggestions don't seem to be keyboard accessible, the down arrow keys after you have typed in a word dont let you select one of the suggestions (see inputTypeahead behavior for example).